__tests__/app.test.js

```diff
@@ -41,4 +42,20 @@ describe('GET /api/articles/:article_id', () => {
             expect(typeof article).toBe('object')
         })
     });
+});
+
+describe('PATCH /api/article/:article_id', () => {
+    test('responds with status: 200 and updated article object', () => {
+        const articleUpdate = {inc_votes: -3}
+        return request(app)
+        .patch('/api/articles/1')
+        .send(articleUpdate)
+        .expect(200)
+        .then((response) => {
+            const article = response.body;
+            console.log(article);
+            expect(typeof article).toBe('object')
```

Comment:
Technically this is a sound assertion, but take a look at a Jest matcher called .toBeInstanceOf - I think you'll find it more robust. (Also, don't forget that typeof [ ] === 'object'...)